doc/src/manual/command-line-options.md

```diff
@@ -29,10 +29,10 @@ The following is a complete list of command-line switches available when launchi
 |`-C`, `--cpu-target <target>`          |Limit usage of CPU features up to `<target>`; set to `help` to see the available options|
 |`-O`, `--optimize={0,1,2,3}`           |Set the optimization level (default level is 2 if unspecified or 3 if used without a level)|
 |`--min-optlevel={0,1,2,3}`             |Set the lower bound on per-module optimization (default is 0)|
-|`-g`, `-g <level>`                     |Enable / Set the level of debug info generation (default level is 1 if unspecified or 2 if used without a level)|
+|`-g`, `-g <level>`                     |Enable or set the level of debug info generation (default level is 1 if unspecified or 2 if used without a level)|
```

Comment:
Should probably update https://github.com/JuliaLang/julia/blob/2ebbb2b3a0b54eae66549bd058ae334cb3642e50/src/jloptions.c#L127 as well.

Reply:
Urghhh, hit an <Enter> accidentally, and d4e21b0 summary becomes incomplete, but you get the point.